streams/readable-byte-streams/general.js

```diff
@@ -16,8 +16,17 @@ test(() => {
 test(() => {
   // Constructing ReadableStream with an empty underlying byte source object as parameter shouldn't throw.
   new ReadableStream({ type: 'bytes' });
+  // Constructor must perform ToString(type).
+  new ReadableStream({ type: { toString() {return 'bytes';} } });
+  new ReadableStream({ type: { toString: null, valueOf() {return 'bytes';} } });
```

Comment:
These don't seem to actually test that ToString happens. To do that you'd need to set a flag and then assert that it was set afterward.